modules/tensor_mechanics/include/materials/RadialReturnStressUpdate.h

```diff
@@ -68,6 +69,10 @@ class RadialReturnStressUpdate : public StressUpdateBase
 
   MaterialProperty<Real> & _effective_inelastic_strain;
   MaterialProperty<Real> & _effective_inelastic_strain_old;
+  Real _max_inelastic_incr;
+
+private:
+  Real _max_inc;
```

Comment:
`private` is absolutely correct and necessary _if_ object integrity needs to be ensured. But in most cases I encounter there is _no_ integrity at stake. It seems to me that this member should just be protected const. This is member is not representing an internal state that needs to be set or read using a particular accessor.